src/views/Items.js

```diff
@@ -0,0 +1,50 @@
+import kind from '@enact/core/kind';
+import Group from '@enact/ui/Group';
+import Item from '@enact/sandstone/Item';
+import Scroller from '@enact/sandstone/Scroller';
+import PropTypes from 'prop-types';
+
+const items = [];
+const url = new URL(window.location.href);
+
+const populateItemsArray = (dataSize) => {
+	for (let i = 0; i < dataSize; i++) {
+		items.push({children: 'Item ' + ('00' + i).slice(-3), id: 'item', key: i});
+	}
+};
+
+const Items = kind({
+	name: 'Items',
+
+	propTypes: {
+		/**
+		*  Number of items in the VirtualList
+		*/
+		dataSize: PropTypes.number
+	},
+
+	defaultProps: {
+		dataSize: 20
+	},
+
+	render: ({dataSize}) => {
+		const urlDataSize = parseInt(url.searchParams.get('dataSize'));
+		let dataSizeProp = dataSize;
+
+		if (!isNaN(urlDataSize)) {
+			dataSizeProp = parseInt(urlDataSize);
+		}
```

Comment:
Why we should call `parseInt` twice?

```suggestion
		const urlDataSize = parseInt(url.searchParams.get('dataSize'));
		const dataSizeProp = urlDataSize  || dataSize || 10;
```
I didn't test it works well or not.

Reply:
Please check the same code on VirtualList.js